src/utils/video-img_2.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+import shutil
+from pathlib import Path
+
+# global variable
+VIDEO_PATH = Path("D:/GDP/TESTING/nofi001.mp4")
+EXTRACT_FOLDER = Path("D:/GDP/TESTING/vi-im/")
+
+def extract_frames(video_path, result_path, index, desired_fps):
+    '''
+        :param video_path: video path
+        :param result_path: the location where the frame image is stored
+        :param index: the starting sequence number of the frame save filename,
+                      this sequence number will be appended to the filename of each saved frame image
```

Comment:
- add a default value to the parameter (0 is a good default value)
- move this parameter after `desired_fps` because it is of lower importance (we will often want our extraction to start at index 0)
- this parameter could even be deleted, but we will discuss about it tomorrow :)